sorted-r-blacklist

```diff
@@ -58,20 +58,16 @@ recipes/bioconductor-bubbletree
 # TODO: bioconductor-cexor needs specific version of genomicranges?
 #recipes/bioconductor-cexor
 
-
+# conflicts with recent Rcpp version
+recipes/r-histonehmm
```

Comment:
can we pin an older version, or do we need to have an older version available for R3.3.2 before :(